ArticleTemplates/liveblogTemplateLivePromo.html

```diff
@@ -118,5 +118,12 @@ <h2 class="live-promo__title">Only got a minute?</h2>
                     fill="#333333" />
             </svg>
         </div>
+        <button class="live-promo__close-button">
```

Comment:
I think we use a cross in an icon font elsewhere. The icons can have blank margins though so this is probably easier to style.

Reply:
I wasn't able to find it. Happy to replace with a `data-icon` specifier if we find what is the code…

Reply:
nevermind, found it!